engine/installation/windows/docker-ee.md

```diff
@@ -26,6 +26,9 @@ on Windows Server 2016 and you have a
 
 ## Install Docker EE
 
+>**Windows Server 1709** Docker Universal Control Plane is not currently supported on Windows Server 1709 due to image incompatibility issues. To use UCP, for now please use the current LTSB Windows release and not 1709.
```

Comment:
@friism so there's a Docker EE 1709 for Windows?

Reply:
No, 1709 is the Windows version. The Docker version would (had it existed) have been `17.09`